packages/origin-ui-core/src/components/DeviceGroupForm.tsx

```diff
@@ -122,6 +123,17 @@ export function DeviceGroupForm(props: IProps) {
 
     const classes = useStyles(useTheme());
 
+    const [files, setFiles] = useState<IUploadedFile[]>([]);
+    const uploadedFiles = files
+        .filter((f) => !f.removed && f.uploadedName)
+        .reduce(
```

Comment:
Perhaps easier and faster way to do this is to use

`.map(x=>x.uploadedName)` 

instead of `.reduce`